contributor_path.rst

```diff
@@ -0,0 +1,214 @@
+
+
+You can find some ideas how you can contribute in the `Contribution guide <contribution_to_project.rst>` and the ``README`` / ``CONTRIBUTING`` files of `corresponding repositories <https://github.com/ansible-collections/>`_.
+
+If the contribution you'd like to give implies technical skills, there can be also quick-start guides which can help.
+
+For example, for collections, you can use the `Quick-start development guide <create_pr_quick_start_guide.rst>`_ to learn how to set up everything you need quickly, test your changes, and submit a pull request.
+
+To make your first experience as smooth as possible, read the repository documentation carefully, then ask the repository maintainers for guidance if you have any questions.
+
+Looking for issues labeled with the ``easyfix``, ``good_first_issue``, and ``docs`` labels can help find good issues to start with.
+
+Continue to contribute
+======================
+
+When you contribute regularly, your proficiency and judgment in the related area will improve quickly and, along with this, the importance of your presence in the project.
+
+Communicate
+===========
+
+Join the channels
+-----------------
+
+Interact and share your ideas with other folks from the community following `Ansible Code of Conduct <https://docs.ansible.com/ansible/latest/community/code_of_conduct.html>`_.
+
+You can find available communication channels in the `Communication guide <https://docs.ansible.com/ansible/devel/community/communication.html>`_.
+
+The most interactive one is on Libera.Chat IRC where many Ansible projects and working groups have dedicated `channels <https://docs.ansible.com/ansible/devel/community/communication.html#irc-channels>`.
+
+IRC meetings
+------------
+
+The important project-scale decisions are made by the community and the Steering Committee at weekly IRC meetings in the ``#ansible-community`` Libera.Chat IRC channel.
+
+If you want to see what is on the agenda, refer to the issues in the `community-topics repository <https://github.com/ansible-community/community-topics>`_. If you want to submit a topic, create an issue in the repository.
+
+The other important Libera.Chat IRC meetings impacting the whole project which you can join are:
+
+  - the Ansible Core IRC meeting in the ``ansible-meeting`` channel.
+  - the Documentation working group IRC meeting in the ``ansible-docs`` channel.
+
+See the `meeting schedule <https://github.com/ansible/community/blob/main/meetings/README.md#schedule>`_.
+
+The Bullhorn newsletter
+-----------------------
+
+Subscribe to `The Bullhorn newsletter <https://docs.ansible.com/ansible/devel/community/communication.html#the-bullhorn>`_ which is released biweekly and contains brief news important for the Ansible developer community.
+
+Contributor summit
+------------------
+
+Take part and meet other contributors in the global quarterly `Ansible Contributor Summit <https://github.com/ansible/community/wiki/Contributor-Summit>`_ virtually or in-person.
```

Comment:
Subscribe to The Bullhorn to know when these take place

Reply:
Added, thanks!